.github/workflows/super-mode-calculator.yaml

```diff
@@ -0,0 +1,67 @@
+name: Upload super-mode-calculator to riff raff
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  super_mode_calculator_build:
+    if: github.repository_owner == 'guardian'
+    # Required by actions-riff-raff
+    permissions:
+      id-token: write
+      contents: read
+
+    name: Build super-mode-calculator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Env
+        run: env
+
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Configure AWS credentials # needed by actions-riff-raff
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - name: Setup Node
+        uses: guardian/actions-setup-node@v2.4.1
+        with:
+          cache: 'yarn'
+          cache-dependency-path: ./super-mode-calculator/yarn.lock
+          node-version: 14.16.0
```

Comment:
Worth noting https://github.com/guardian/actions-setup-node is deprecated (and archived) in favour of its upstream fork https://github.com/actions/setup-node.

Thoughts on how to make this easier to discover, i.e. reduce the chance/risk of the deprecated action being used, most welcomed.